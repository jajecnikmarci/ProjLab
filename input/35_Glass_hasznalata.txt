timercontrol enable
starttest
add room room1 3
add room room2 3
add door room1 room2 door1
add item room1 Glass glass
add person room1 Student student1
add person room2 Professor professor1
pickup student1
printstate student1
gotoroom professor1 room1
printstate student1
gotoroom professor1 room2
endtimer timer1
printstate student1
endtest