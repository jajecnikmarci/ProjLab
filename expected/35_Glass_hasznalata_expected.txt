student1
	- location: room2
	- stunned: false
	- inventory: 
		glass
	- killImmunities: 
		killImmunity1
	- poisonImmunities:
	- souls: 3
student1
	- location: room2
	- stunned: false
	- inventory: 
		glass
	- killImmunities: 
		killImmunity1
	- poisonImmunities:
		poisonImmunity2
	- souls: 3
student1
	- location: room2
	- stunned: true
	- inventory: 
	- killImmunities: 
	- poisonImmunities:
	- souls: 3